openedx/core/djangoapps/user_authn/views/password_reset.py

```diff
@@ -0,0 +1,114 @@
+""" Handles password resets logic. """
```

Comment:
Kudos on keeping the various logically-separated views in their own separate python modules!